libkbfs/prefetcher.go

```diff
@@ -165,6 +174,10 @@ func (p *blockPrefetcher) newPrefetch(count int, triggered bool,
 		ctx:               ctx,
 		cancel:            cancel,
 		waitCh:            make(chan struct{}),
+		PrefetchProgress: PrefetchProgress{
+			SubtreeBytesTotal: bytes,
+			Start:             p.config.Clock().Now(),
```

Comment:
The tests often need to fake out the time, so we always use an interface for the clock instead of directly using `time`, to have more control over it during testing.